### ISSUE TITLE
[Config][Bug?] Explore existing Husky hook with git ammend

### ISSUE THREAD

**Issue by kat-kan** [kat-kan/kawqa-gad-playwright#40]
This is a low priority, but I want to have it written down somewhere.
I was working on branch with correct name (tests/ISSUE-21_add_post_articles_test) and I wanted to edit message for one of the commits. 
I was blocked by Husky validation even though the commit was previously accepted and I was on the same branch.

Scope of the issue:
- test git amend with our Husky config
- if bug confirmed, look for solution or at least create a new issue for it.
